### ISSUE TITLE
Space at the end of the className

### ISSUE THREAD

**Issue by diegoemerique** [negomi/react-burger-menu#211]
I have a snapshot test, that renders like this before: 
```
<span
       className="bm-cross"
```

And after I updated the "react-burger-menu" package, the snapshot fails, and returns like this:
```
<span
       className="bm-cross "
```
* There is an extra space at the end of the string.

Somebody knows how can I can fix it? or maybe if we can fix it adding a trim() to the project.

Thanks,
and sorry if I'm not supposed to ask it here. Or if it is not a problem and I miss something.  :)

**Comment by negomi** [comment]
Hi @diegoemerique, thanks for the issue! This wasn't a problem for general usage but I can see how it might mess up your snapshot tests :)

Fixed in `2.2.0`

**Comment by diegoemerique** [comment]
Thank you!
